metrics-delivery-endpoint-impl/pom.xml

```diff
@@ -30,11 +30,6 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
-        
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-devtools</artifactId>
-        </dependency>
```

Comment:
We might want to actually keep it for the automatic restarts after recompilation?
Just set it to optional [as the docs suggest](https://docs.spring.io/spring-boot/docs/2.5.4/reference/htmlsingle/#using.devtools).

Although I'm still not sure how useful it really is during development.

Reply:
OK, as discussed during the call, we ain't gonna need it ;)